features/admin_manages_invoices.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Admin creates invoice
+  As GigaFood
+  In order to get payment for the order
+  I need to generate a PDF invoice with order details and payment information
+
+
+  Background:
+    Given the following categories exist:
+      | name    | description                |
+      | Starter | Description for category 1 |
+
+    And the following dishes exist:
+      | name   | description            | price | min_quantity | category |
+      | Dish 1 | Description for Dish 1 | 100   | 10           | Starter  |
+      | Dish 2 | Description for Dish 2 | 200   | 10           | Starter  |
+      | Dish 3 | Description for Dish 3 | 300   | 10           | Starter  |
+
+    And the following orders exist:
```

Comment:
Don't know if it matters, but in the database we have two separate columns for time and date.